### PR TITLE
[new] [author] Adding sp-pnp-js w/ npm autoupdate

### PR DESCRIPTION
**Pull request by patrick-rodgers**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/OfficeDev/PnP-JS-Core
 * Official website (optional, not the repository): https://dev.office.com/sharepoint
 * NPM package url (optional): https://www.npmjs.com/package/sp-pnp-js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 83
   - Count of watchers: 48
   - Count of forks: 73
 * NPM download stats (optional):
   - Downloads in the last day: 45
   - Downloads in the last week: 209
   - Downloads in the last month: 648

# Essential checklist
 * [X ] I'm the author of this library
 * [ X] I would like to add link to the page of this library on CDNJS on website / readme
 * X[ ] This lib was not found on cdnjs repo
 * [X ] No already exist / duplicated issue and PR
 * [X ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X ] More than 500 downloads stats per month on npm registry
 * [X ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X ] Has valid tags for each versions (for git auto-update)
 * [X ] Auto-update setup
 * [X ] Auto-update target/source is valid.
 * [X ] Auto-update filemap is correct.

# Git commit checklist
 * [X ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ X] Separate unrelated changes into different commits.
 * [X ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X ] Close corresponding issue in commit message
 * [X ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/sp-pnp-js/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "spfx",
+    "sharepoint framework"
+  ],
+  "npmName": "sp-pnp-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "pnp.js",
+        "pnp.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/sp-pnp-js/package.json]: What about `pnp-provisioning.min.js` and other map files?

Comment (PeterDaveHello) [inline ajax/libs/sp-pnp-js/package.json]: I think we could use a single rule `"**/!(*.ts)"` here

**Comment by patrick-rodgers** [comment]
Thanks for the feedback, working on this

**Comment by PeterDaveHello** [comment]
@patrick-rodgers Thanks!

**Comment by x09326** [comment]
@patrick-rodgers 
Hi, I think you can take [this](https://dev.office.com/sharepoint) as the `homepage` field in the *package.json*.

**Comment by PeterDaveHello** [comment]
@x09326 I don't so, nothing directly talk about the pnp-js

**Comment by patrick-rodgers** [comment]
Did the needed updates. We don't really have a homepage beyond the git readme currently, but the dev.office.com/sharepoint will include some details about the library but never be only about it.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by patrick-rodgers** [comment]
Thanks all for the help getting this listed!

**Pull request merged**